stewart/parts/header-title-nav.html

```diff
@@ -1,6 +1,11 @@
-<!-- wp:group {"align":"full","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"top":"var(--wp--custom--margin--horizontal, 30px)","bottom":"40px"}}}} -->
-<div class="wp-block-group alignfull" style="padding-top:var(--wp--custom--margin--horizontal, 30px);padding-bottom:40px">
-	<!-- wp:site-title /-->
-	<!-- wp:navigation {"isResponsive":true,"__unstableLocation":"primary"} /-->
-</div>
+<!-- wp:group {"align":"full","layout":{"inherit":true}} -->
+<div class="wp-block-group alignfull"><!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var(-d-dwp-d-dcustom-d-dmargin-d-dhorizontal, 30px)","bottom":"40px"}}},"layout":{"type":"flex","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--custom--margin--horizontal, 30px);padding-bottom:40px"><!-- wp:group {"layout":{"type":"flex"}} -->
+<div class="wp-block-group"><!-- wp:site-logo {"width":64} /-->
+
+<!-- wp:site-title /--></div>
+<!-- /wp:group -->
+
+<!-- wp:navigation {"__unstableLocation":"primary","layout":{"type":"flex","orientation":"horizontal"}} /--></div>
+<!-- /wp:group --></div>
```

Comment:
This template was edited to ensure that this version of the header is wide-aligned, like the rest of the site content.